README.md

```diff
@@ -10,13 +10,35 @@ The members of Full Stack Embedded 2016 are, in alphabetical order:
 
 * Frederic Afadjigla
 * Nico Caspari
+* William Herndon
 * Daniel Lee
-* Rüdiger Reiter
+* Ruediger Reiter
 * Gregor Schnee
 
 This component was authored mainly by [Daniel Lee](erget2005@gmail.com) and 
 [Gregor Schnee](schneegor@gmail.com).
 
+## Required/suggested packages apt-get packages
+Use "sudo apt-get install <package-name>[, <package-name>]"
+### Required:
+* python3
+* python3-pip (required for setup.py to run)
+
+## Suggested for development setup:
```

Comment:
:) alright, I've reworked quite a bit now in ``README`` and will send you a pull request for review in a few minutes.

Django is really needed, correct, but as long as ``pip`` is installed it can be installed by ``setup.py``. Since we've listed ``pip`` as a dependency that's all that's needed to get up and running. Listing it separately in the ``setup.py`` makes it machine readable and available according to the "normal" Python conventions, where a Python person would look to find the dependencies, and if you're not a dev you probably are just wanting to install the package anyway. For that reason I'd rather just avoid the clutter and leave the dependency list there.